### PR TITLE
Implement inclusive writing for admin and channel op

### PR DESCRIPTION
**Pull request by bear101**

### CODE REVIEW

**Comment by bear101** [comment]
@CoBC it looks ok because it preserves the existing translations:
`     <message>
         <location filename="../channelstree.cpp" line="984"/>
         <source>Administrator</source>
        <comment>For female</comment>
        <translation type="unfinished">Администратор</translation>
    </message>
    <message>
        <location filename="../channelstree.cpp" line="984"/>
        <source>Administrator</source>
        <comment>For male and neutral</comment>
         <translation type="unfinished">Администратор</translation>
     </message>`
So go ahead and update other places you like

**Comment by amirmahdifard** [comment]
hello, please explane to me what this request will do, even after i red the changed files.

**Comment by CoBC** [comment]
In some languages including French, Spanish, Portuguese, Italian, 
Swidish and more, some words like "administrator" or "operator" have 
different writings according to gender of people concern, so this PR 
allow translators to take this in count in translations and the client 
will shown correct writing according to user gender. For now I only 
implement this where it's not to difficult to do it, because some 
sentences should be completely rewrite to add this but it requires a lot 
of work for translators so not sure for now...

**Comment by amirmahdifard** [comment]
oh, thanks for explaning.

**Comment by CoBC** [comment]
@bear101 I can't request a review from you, but you can review when you're ready.

File: Client/qtTeamTalk/mainwindow.cpp [hunk 1]
```diff
@@ -1135,9 +1135,9 @@ void MainWindow::clienteventCmdUserLoggedOut(const User& user)
     m_textmessages.clearUserTextMessages(user.nUserID);
     if (user.nUserID != TT_GetMyUserID(ttInst))
     {
-        addStatusMsg(STATUSBAR_USER_LOGGEDOUT, tr("%1 has logged out") .arg(getDisplayName(user)));
+        addStatusMsg(STATUSBAR_USER_LOGGEDOUT, ((user.nStatusMode & STATUSMODE_FEMALE)?tr("%1 has logged out", "For female").arg(getDisplayName(user)):tr("%1 has logged out", "For male and neutral").arg(getDisplayName(user))));
         playSoundEvent(SOUNDEVENT_USERLOGGEDOUT);
-        addTextToSpeechMessage(TTS_USER_LOGGEDOUT, QString(tr("%1 has logged out") .arg(getDisplayName(user))));
+        addTextToSpeechMessage(TTS_USER_LOGGEDOUT, QString(((user.nStatusMode & STATUSMODE_FEMALE)?tr("%1 has logged in", "For female").arg(getDisplayName(user)):tr("%1 has logged in", "For male and neutral").arg(getDisplayName(user)))));
```
Comment (bear101) [inline Client/qtTeamTalk/mainwindow.cpp]: Looks incorrect. Shouldn't it say "%1 has logged out"?

File: Client/qtTeamTalk/mainwindow.cpp [hunk 2]
```diff
@@ -1137,7 +1137,7 @@ void MainWindow::clienteventCmdUserLoggedOut(const User& user)
     {
         addStatusMsg(STATUSBAR_USER_LOGGEDOUT, ((user.nStatusMode & STATUSMODE_FEMALE)?tr("%1 has logged out", "For female").arg(getDisplayName(user)):tr("%1 has logged out", "For male and neutral").arg(getDisplayName(user))));
         playSoundEvent(SOUNDEVENT_USERLOGGEDOUT);
-        addTextToSpeechMessage(TTS_USER_LOGGEDOUT, QString(((user.nStatusMode & STATUSMODE_FEMALE)?tr("%1 has logged in", "For female").arg(getDisplayName(user)):tr("%1 has logged in", "For male and neutral").arg(getDisplayName(user)))));
+        addTextToSpeechMessage(TTS_USER_LOGGEDOUT, QString(((user.nStatusMode & STATUSMODE_FEMALE)?tr("%1 has logged out", "For female").arg(getDisplayName(user)):tr("%1 has logged out", "For male and neutral").arg(getDisplayName(user)))));
```
Comment (bear101) [inline Client/qtTeamTalk/mainwindow.cpp:1140]: Ah, it's corrected here. I'll squash the commits.

**Pull request merged**